bool enable
float64[] desired
---
bool status